src/Fixer/PhpBasic/Feature/VisibilityPropertiesFixer.php

```diff
@@ -293,26 +292,53 @@ private function getPropertyVariable(Tokens $tokens, int $key, bool $propertyExc
         return null;
     }
 
-    private function insertVariablePropertyWarning(Tokens $tokens, int $key, string $convention)
+    private function insertVariablePropertyWarning(Tokens $tokens, int $key, string $convention): void
     {
         $this->insertComment(
             $tokens,
-            $tokens->getNextTokenOfKind($key, [';']),
+            $key,
             $tokens[$key]->getContent(),
             $convention,
         );
     }
 
-    private function insertComment(Tokens $tokens, int $insertIndex, string $propertyName, string $convention)
+    private function insertComment(Tokens $tokens, int $insertIndex, string $propertyName, string $convention): void
     {
         $comment = '// TODO: "' . $propertyName . '" - ' . $convention;
-        if (!$tokens[$tokens->getNextNonWhitespace($insertIndex)]->isGivenKind(T_COMMENT)) {
+
+        do {
+            $token = $tokens[$insertIndex++];
+        } while (!substr_count($token->getContent(), "\n"));
+
+        if (!$tokens[$tokens->getNextNonWhitespace($insertIndex - 3)]->isGivenKind(T_COMMENT)) {
             $tokens->insertSlices([
-                ++$insertIndex => [
+                $insertIndex - 1 => [
                     new Token([T_WHITESPACE, ' ']),
                     new Token([T_COMMENT, $comment]),
                 ],
             ]);
         }
     }
+
+    /**
+     * @param array<Token> $previousTokens
+     * @return Token|null
+     */
+    private function getVisibilityToken(array $previousTokens): ?Token
+    {
+        $visibilityToken = null;
```

Comment:
this one is redundant, just return right away
```
private function getVisibilityToken(array $previousTokens): ?Token
{
    foreach ($previousTokens as $previousToken) {
        if ($previousToken->isGivenKind([T_PUBLIC, T_PROTECTED, T_PRIVATE, 10028, 10029, 10030])) {
            return $previousToken;
        }

        if ($previousToken->isGivenKind(T_FUNCTION)) {
            return null;
        }
    }

    return null;
}

```

Reply:
Agree. Fixed.